landavailability/api/management/commands/import_addresses.py

```diff
@@ -0,0 +1,41 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+from api.models import Address
+import csv
+
+
+class Command(BaseCommand):
+    help = 'Import addresses from a CSV file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('csv_file', type=str)
+
+    def handle(self, *args, **options):
+        csv_file_name = options.get('csv_file')
+
+        if csv_file_name:
+            with open(csv_file_name, newline='') as csvfile:
```

Comment:
Good idea! Until line 20 it's basically the same for all the scripts. As soon as I have the second script, I will refactor them with a CSVImportCommand base class.